src/cryptonote_basic/cryptonote_basic_impl.cpp

```diff
@@ -95,24 +95,35 @@ namespace cryptonote {
   uint64_t block_reward_unpenalized_formula_v8(uint64_t height)
   {
     std::fesetround(FE_TONEAREST);
-    uint64_t result = 28000000000.0 + 100000000000.0 / loki::exp2(height / (720.0 * 90.0)); // halve every 90 days.
+    uint64_t result = 28'000'000'000. + 100'000'000'000. / loki::exp2(height / (720. * 90)); // halve every 90 days.
     return result;
   }
 
+  constexpr uint64_t block_reward_unpenalized_formula_v15() {
```

Comment:
These 2 functions v15, v16 look to be unused. Below you use the values directly which is fine.

Reply:
Whoops, yeah, changed my mind about using functions and forgot to remove them.  Gone now.